packages/api/tests/endpoints/test-rules.js

```diff
@@ -268,16 +361,54 @@ test('POST creates a rule', async (t) => {
     .expect(200);
 
   const { message, record } = response.body;
+  const fetchedDynamoRecord = await ruleModel.get({
+    name: newRule.name,
+  });
+
+  const fetchedPostgresRecord = await t.context.dbClient.queryBuilder()
+    .select()
+    .table(tableNames.rules)
+    .where({ name: newRule.name })
+    .first();
+
   t.is(message, 'Record saved');
 
   newRule.createdAt = record.createdAt;
   newRule.updatedAt = record.updatedAt;
 
-  t.deepEqual(record, newRule);
+  t.is(fetchedPostgresRecord.collection_cumulus_id, collectionCumulusId);
+  t.is(fetchedPostgresRecord.provider_cumulus_id, providerCumulusId);
+
+  t.deepEqual(
+    omit(fetchedPostgresRecord, ['cumulus_id', 'collection_cumulus_id', 'provider_cumulus_id']),
```

Comment:
Nit: I would just add the `collection_cumulus_id` and `provider_cumulus_id` to the deepEqual comparison